docs/Homebrew-and-Python.md

```diff
@@ -2,18 +2,16 @@
 
 This page describes how Python is handled in Homebrew for users. See [Python for Formula Authors](Python-for-Formula-Authors.md) for advice on writing formulae to install packages written in Python.
 
-Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python) and defaults to the macOS system Python.
-
-Homebrew provides formulae to brew Python 3.y. A `python@2` formula was provided until the end of 2019, at which point it was removed due to the Python 2 deprecation.
+Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python) and defaults to the macOS system Python. On Linux Homebrew will install the necessary Python version that is needed to make your packages work.
```

Comment:
```suggestion
Homebrew should work with any [CPython](https://stackoverflow.com/questions/2324208/is-there-any-difference-between-cpython-and-python). On Linux Homebrew will install the necessary Python version that is needed to make your packages work.
```

macOS system Python doesn't exist anymore and I suspect "defaults to" doesn't really make any sense here anymore (it's not keg-only anymore).

Reply:
```suggestion
```

Or to be honest, the whole sentence doesn't even make sense. If "Homebrew should work with" means what formulae will pick up at build-time, then we don't allow arbitrary installations of Python anymore and haven't for quite some time under the superenv so "any CPython" isn't true.

Reply:
I have cleanup up this part